discovery.md

```diff
@@ -0,0 +1,16 @@
+# Box-client discovery: two options
+
+## Option 1: --no-cloud
```

Comment:
Right, maybe it's more correct to call it a runtime option, because as a developer, you could set it from the command line.

However, to the end-user it would be a "purchase time" option unless a hardware developer adds some UI to the box (like a big red button) that affects how `cargo run` is called when you power up the box.

I'll have a stab today/tomorrow at implementing this command line option.